src/plugins/merge.ts

```diff
@@ -41,20 +56,26 @@ export class Merge extends ManifestPlugin {
     const releaseData: ReleaseData[] = [];
     const labels = new Set<string>();
     let rawUpdates: Update[] = [];
+    let rootRelease: CandidateReleasePullRequest | null = null;
     for (const candidate of candidates) {
       const pullRequest = candidate.pullRequest;
       rawUpdates = rawUpdates.concat(...pullRequest.updates);
       for (const label of pullRequest.labels) {
         labels.add(label);
       }
       releaseData.push(...pullRequest.body.releaseData);
+      if (candidate.path === '.') {
+        rootRelease = candidate;
+      }
     }
     const updates = mergeUpdates(rawUpdates);
 
     const pullRequest = {
-      title: PullRequestTitle.ofTargetBranch(
+      title: PullRequestTitle.ofComponentTargetBranchVersion(
+        rootRelease?.pullRequest.title.component,
         this.targetBranch,
-        MANIFEST_PULL_REQUEST_TITLE_PATTERN
+        rootRelease?.pullRequest.title.version,
```

Comment:
If we found a root package release pr, we'll "hoist" its component and version args to this title pattern